packages/analytics-js-common/src/utilities/checks.ts

```diff
@@ -64,6 +64,13 @@ const isDefinedNotNullAndNotEmptyString = (value: any): boolean =>
  */
 const isTypeOfError = (obj: any): obj is Error => obj instanceof Error;
 
+/**
+ * Determines if the SDK is running inside a chrome extension
+ * @returns boolean
+ */
+const isSDKRunningInChromeExtension = (): boolean =>
```

Comment:
This method is more suited for the capabilities detection utils, lets split in a new file please

Reply:
dhecks are for value/type checks

Reply:
Done